scripts/splunkscript/readme.md

```diff
@@ -0,0 +1,68 @@
+﻿# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+
+## Steps to Configure & run Splunk compliance test.
+
+Pre-requiste:
+
+Splunk s3test should be installed and configured on the system.
+
+Steps:
+
+1. Virtualenv should be installed on the system
+   yum install python-virtualenv
```

Comment:
convert this in blockquote. You can refer other readme files in s3server repo.

Reply:
Its already there in blockquote.
https://github.com/Seagate/cortx-s3server/blob/br/shazia/ceph_splunk_automation/scripts/splunkscript/readme.md